dqgen/adapters/ap_reader.py

```diff
@@ -11,7 +11,7 @@
 """
     This module is responsible for reading an application profile (for diffing) from a CSV file.
 """
-NA_VALUES = {"class": "", "property": "", "object property": "", "modifiable": 0, "language dependent": 0}
+NA_VALUES = {"class": "", "property": "", "object property": "", "preview property": "", "preview object property": ""}
```

Comment:
oh, you decided to do it here? 
I thought you will do that in the template! Whatever works!

Reply:
I wll do it in template...this is something else